### ISSUE TITLE
aria description for downoad link in image gallery

### ISSUE THREAD

**Issue by jackahl** [kitconcept/volto-light-theme#412]
The current link in the image gallery to download the image is not making clear to screenreaders that it is a download. This should be improved.
